Source/Models/Highlight+Helper.swift

```diff
@@ -150,6 +150,7 @@ extension Highlight {
             highlight?.type = type.hashValue
             
             try realm.commitWrite()
```

Comment:
Maybe it is a good idea to use `guard let` to create a `Realm`

Maybe changing `realm.beginWrite()` to `realm.write { }` will be better. Will prevent screen locking.